components/Navigation/Navigation.js

```diff
@@ -42,8 +42,12 @@ export const Navigation = () => {
       </div>
       <nav
         className={classNames(
-          { hidden: !isMenuOpen },
-          'bg-accent-200 absolute top-[52px] md:top-[76px] right-0 z-50 lg:bg-transparent lg:relative lg:flex-1 lg:flex lg:top-0',
+          {
+            hidden: !isMenuOpen,
+            'lg:text-black': isScrolledDown,
+            'lg:text-white': !isScrolledDown,
+          },
+          'bg-accent-200 absolute top-[52px] md:top-[60px] right-0 z-50 text-black lg:bg-transparent lg:relative lg:flex-1 lg:flex lg:top-0',
```

Comment:
I do strongly recommend create a constat with 3 levels of z-index

```
const zIndex =  {
  baseLevel: 'z-0',
  foreground: 'z-10',
  modal: 'z-20'
 }
```


or something alongside that :D. Shooting straight for z-index: 50 is possible for something like modal f.e. But let's be very careful with those. I would predefine them.



![552c2c06f7fd1b9e94d70f2e7622cfc0](https://user-images.githubusercontent.com/12488648/161928584-6f3c23a3-e193-4881-ad10-302b2569f798.jpeg)

Reply:
:)